### ISSUE TITLE
Cannot autowire service "Frosh\Rector\Rule\v65\RedisConnectionFactoryCreateRector"

### ISSUE THREAD

**Issue by pietrzakadrian** [FriendsOfShopware/shopware-rector#21]
### PHP Version

8.1

### Shopware Version

6.5

### Actual behaviour

[ERROR] Cannot autowire service "Frosh\Rector\Rule\v65\RedisConnectionFactoryCreateRector": argument
         "$nodesToAddCollector" of method "__construct()" has type "Rector\PostRector\Collector\NodesToAddCollector" but
         this class was not found.

### Expected behaviour

. 

### Steps to Reproduce?

./vendor/bin/rector process src

**Comment by zaifastafa** [comment]
@shyim I am getting the same error

```
 [ERROR] Cannot autowire service "Frosh\Rector\Rule\v65\RedisConnectionFactoryCreateRector": argument                   
         "$nodesToAddCollector" of method "__construct()" has type "Rector\PostRector\Collector\NodesToAddCollector" but
         this class was not found.    
```

**Comment by pietrzakadrian** [comment]
use rector 0.15.7 instead of 0.17.2

"rector/rector": "~0.15.7",

**Comment by shyim** [comment]
Fixed by 0.2.0
